_rules/6cbcbe.md

```diff
@@ -0,0 +1,206 @@
+---
+name: Links with identical accessible names serve equivalent purpose
+test_type: atomic
+
+description: |
+  This rule checks that links with identical accessible names serve an equivalent purpose.
+
+success_criterion: 
+- 2.4.9 # Link Purpose (Link Only)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test Procedure
+
+### Applicability
+
+This rule applies to any two or more HTML or SVG elements that have the [semantic role](#semantic-role) of link, are [included in the accessibility tree](#included-in-the-accessibility-tree), and that have the same [non-empty](#non-empty) [accessible name](#accessible-name).
+
+### Expectation
+
+When activated, the links in each set of target elements resolve to the [same resource](#same-resource) or to different resources that fulfill an [equivalent purpose](#equivalent-purpose).
+
+**Note:** Resolving the links includes potential redirects.
+
+## Assumptions
+
+This rule assumes that the purpose of the links for links with identical names would not be ambiguous to users in general.
```

Comment:
You could simplify "the purpose of the links for links with identical names" to "the purpose of the links with identical names"

Reply:
I'm not sure I understand what this means. Can you add to it to make this clearer?